src/components/layout/styles/Layout.scss

```diff
@@ -0,0 +1,23 @@
+@import "~@edx/paragon/scss/edx/theme.scss";
+@import "~@edx/paragon/scss/edx/fonts.scss"; // Roboto
+
+$fa-font-path: "~font-awesome/fonts";
+@import "~font-awesome/scss/font-awesome";
```

Comment:
If these files are already imported in src/index.scss, do they need to be here?

Reply:
Got rid of the ones that weren't needed 👍